nearlib/test/integration.test.js

```diff
@@ -1,14 +1,50 @@
-const account = require('../account');
-
-const aliceAccount = {
-    account_id: 'alice.near',
-    amount: 100,
-    stake: 0,
-    nonce: 0,
-    code_hash: '2Kkfab2QQ3GK7DSiuzcRF2H4vNSPKFfE9gkCDkPGumtV'
+//import Account from '../account';
+
+const test_key_store = require('../test-tools/in_memory_key_store.js');
+const Account = require('../account');
+const aliceAccountName = 'alice.near';
+const aliceKey = {
+    public_key: "9AhWenZ3JddamBoyMqnTbp7yVbRuvqAv3zwfrWgfVRJE",
+    secret_key: "2hoLMP9X2Vsvib2t4F1fkZHpFd6fHLr5q7eqGroRoNqdBKcPja2jCrmxW9uGBLXdTnbtZYibWe4NoFtB4Bk7LWg6"
 };
+test_key_store.setKey(aliceAccountName, aliceKey);
+const account = new Account(test_key_store);
+const TEST_MAX_RETRIES = 3;
```

Comment:
```
const TEST_MAX_RETRIES = parseInt(process.env.TEST_MAX_RETRIES || "3");
```

you might have much easier time tinkering with CI if tweakable parameters can be set in env vars